src/commands/login.ts

```diff
@@ -20,23 +21,40 @@ export default class Login extends Command {
       const { auth } = config
       try {
         await api.checkAuth(auth)
-        this.log(`User ${config.username} is already logged in. To log in with a different user, run jamsocket logout first.`)
+        this.log('You are already logged in. To log in with a different token, run jamsocket logout first.')
```

Comment:
With token-based login, we unfortunately don't know what account the token belongs to. We could expose an endpoint that returns this (or return it from the auth check endpoint), but I'm unsure if it's a good idea to let someone query what account an API token belongs to. Seems safer to allow users to see all API tokens for their account and let them find a match that way.

Reply:
Yeah, that's a good idea